src/setup/index.ts

```diff
@@ -13,7 +14,7 @@ if (!TARGET_TOKEN || !TARGET_CLIENT_ID) {
 
 const rest = new REST({ version: '9' }).setToken(TARGET_TOKEN);
 
-function commands(): object[] {
+function commands(): ApplicationCommandData[] {
```

Comment:
```suggestion
function commands(): RESTPostAPIApplicationCommandsJSONBody[] {
```

多分こっちの型が正しいやつかも... (ごめん)